go.mod

```diff
@@ -3,9 +3,9 @@ module github.com/aquasecurity/trivy
 go 1.13
 
 require (
-	github.com/aquasecurity/fanal v0.0.0-20200413182139-9213b79eba1a
+	github.com/aquasecurity/fanal v0.0.0-20200427221647-c3528846e21c
 	github.com/aquasecurity/go-dep-parser v0.0.0-20190819075924-ea223f0ef24b
-	github.com/aquasecurity/trivy-db v0.0.0-20200408191531-0a25a37ec16f
+	github.com/aquasecurity/trivy-db v0.0.0-20200427223346-73deea114f99
```

Comment:
This should be updated to point to master once https://github.com/aquasecurity/trivy-db/pull/42 is merged.

Reply:
I fixed it.
https://github.com/aquasecurity/trivy/pull/476/commits/10ab9ac8de03b952b06c4ee84a13c646df48dba0